libs/portal/pages/schedule.tsx

```
@@ -0,0 +1,69 @@
+            SCHEDULE
+          </Typography>
+        </GlassmorphicCard>
+        <div className="flex flex-col gap-5">
+
+          <div className="flex flex-row gap-3 justify-center ">
+            <Button className="!hover:text-white cursor-pointer">Fri Mar 14th</Button>
+            <Button className="cursor-pointer">Fri Mar 14th</Button>
+            <Button className="cursor-pointer">Fri Mar 14th</Button>
+          </div>
+          <MultipleSelector isRequired={false} name="Filter" form={undefined} options={options} placeholder="filter" hidePlaceholderWhenSelected></MultipleSelector>
+          <Event />
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)